docker-hub/company-owner.md

```diff
@@ -4,6 +4,11 @@ keywords: company, owners
 title: Manage company owners
 ---
 
+> **Note**
+>
+> Company layer is an [early access](../release-lifecycle.md#early-access-ea)
```

Comment:
Are we calling this feature "Company layer"? If so, should we also update https://docs.docker.com/docker-hub/creating-companies?

Reply:
Checked with Mercedes and 'company layer' is approved. I don't think we need to update the company overview page though.